pkg/executables/tink.go

```diff
@@ -0,0 +1,41 @@
+package executables
+
+import (
+	"context"
+	"fmt"
+)
+
+const (
+	tinkPath                   = "tink"
+	TinkerbellCertUrlKey       = "TINKERBELL_CERT_URL"
+	TinkerbellGrpcAuthorityKey = "TINKERBELL_GRPC_AUTHORITY"
+)
+
+type Tink struct {
+	Executable
+	tinkerbellCertUrl       string
+	tinkerbellGrpcAuthority string
+}
+
+func NewTink(executable Executable, tinkerbellCertUrl, tinkerbellGrpcAuthority string) *Tink {
+	return &Tink{
+		Executable:              executable,
+		tinkerbellCertUrl:       tinkerbellCertUrl,
+		tinkerbellGrpcAuthority: tinkerbellGrpcAuthority,
+	}
+}
+
+func (t *Tink) getEnvMap() map[string]string {
+	return map[string]string{
```

Comment:
nit: you could store this in the `Tink` struct and avoid building it on every command run